src/AuthApiClient.ts

```diff
@@ -0,0 +1,27 @@
+import { AuthorizedApiClient } from './AuthorizedApiClient';
+import { FetchOptions, HTTP_METHOD, RequestBody } from './ApiClient';
+
+class AuthApiClient extends AuthorizedApiClient {
+  protected entity;
+
+  constructor(
+    baseUrl: string,
+    clientId: string,
+    clientSecret: string,
+    entity: 'borrowers' | 'intermediaries',
```

Comment:
you can do like "protected entity: 'borrowers' | 'intermediaries'," and you don't need "protected entity"/"this.entity = entity;"

Reply:
✔️